fairroot/base/sim/FairModule.cxx

```diff
@@ -280,8 +280,9 @@ void FairModule::AddSensitiveVolume(TGeoVolume* vol)
     auto volName = vol->GetName();
     LOG(debug2) << "AddSensitiveVolume " << volName;
 
-    auto addedVol = vList->addVolume(std::make_unique<FairVolume>(volName, fNbOfVolumes));
+    auto addedVol = vList->addVolume(std::make_unique<FairVolume>(volName, fNbOfVolumes, 0, this));
```

Comment:
1. Please improve the other calls to `vList->addVolume` as well. So that they also have the proper values set.

2. If you call the extended ctor, then plasse set the modid correctly as well.

   ```suggestion
       auto addedVol = vList->addVolume(std::make_unique<FairVolume>(volName, fNbOfVolumes, fModId, this));
   ```

3. If you have done this, consider dropping the first two lines of `FairModule::RegisterSensitiveVolume`.